src/dialogs/more-info/controls/more-info-vacuum.js

```diff
@@ -41,9 +41,20 @@ class MoreInfoVacuum extends PolymerElement {
       <p></p>
       <div class="status-subtitle">Vacuum cleaner commands:</div>
       <div class="horizontal justified layout">
-        <div hidden$="[[!supportsPause(stateObj)]]">
-          <paper-icon-button icon="hass:play-pause" on-click="onPlayPause" title="Start/Pause"></paper-icon-button>
-        </div>
+        <template is="dom-if" if="[[supportsStart(stateObj)]]">
+          <div hidden$="[[!supportsStart(stateObj)]]">
```

Comment:
Why have this hidden attribute if you're already inside a dom-if checking for the opposite?